### DISCUSSION TITLE
Error: .github#L1, specified action is not allowed to be used in my organization

### DISCUSSION THREAD

**Discussion by ElonKim-71** [community/community#36535] [Actions]
I would like to use relekang/python-semantic-release github actions. 
First I tried to use it in terminal and git actions in own private github and work well.

However, in my organization, I would like to use the same actions, but it didn't work. 
Here is annotations. 
![image](https://user-images.githubusercontent.com/71682338/196395571-f5242b29-e697-4225-87a7-e45daee0ef3c.png)

**Comment by airtower-luna** [comment]
It seems your organization limits which actions may be called from workflows within the organization, and the error message says that the specific action you're trying to use isn't allowed. See [Managing GitHub Actions permissions for your organization](https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization#managing-github-actions-permissions-for-your-organization) for the settings.

You'll either need to convince an organization admin to allow that action (or do it yourself, if you're an admin), or find a way to achieve what you want without that action.

**Reply by ElonKim-71** [reply]
Thank you, but I already tried to that permissions process after see the guide pages.
![image](https://user-images.githubusercontent.com/71682338/196597612-cb0c1caf-954e-4723-9905-ca15e707c741.png)

Here is my settings.
![image](https://user-images.githubusercontent.com/71682338/196597815-536a00ee-e46d-499d-8b1c-66de9bee9d64.png)

**Reply by airtower-luna** [reply]
That looks about right. The only other thing I can think of right now: Check if there are more restrictive settings in the repository itself. :thinking:

**Comment by ossanna16** [comment]
Hey there @ElonKim-71 and thank you for asking a good question! I'm glad you found our community 🙂
